client/src/components/UserProfile/Tiles/EventTile.jsx

```diff
@@ -4,19 +4,22 @@ import BookEvent from '../BookingModals/BookEvent.jsx';
 
 import BookHotel from '../BookingModals/BookHotel.jsx';
 import {
-  Grid,
+  Grid, Button,
 } from '@material-ui/core';
 
-export default function EventTile({ eventInfo, eventBookInfo}) {
+export default function EventTile({ eventInfo, eventBookInfo, handleEventDelete }) {
   // const classes = useStyles();
-  console.log('event info', eventInfo);
-  console.log('eventBookInfo', eventBookInfo);
 
   return (
     <div>
       <div>
-        {eventInfo.name}
+        {
+        eventInfo ? eventInfo.name : null
+        }
```

Comment:
eventinfo should never be null because of checks in upper scope

Reply:
Getting heebie jeebies so I'm just going to leave it...